examples/binding-patterns-in-assignment/binding_patterns_in_assignment.bal

```diff
@@ -0,0 +1,30 @@
+import ballerina/io;
+
+int x = 0;
+int y = 1;
+
+type IntPair [int, int];
+
+function assign(IntPair ip) {
+    // The variable `ip` of type `IntPair` is destructured and
+    // assigned to the two module-level variables `x` and `y`.
+    [x, y] = ip;
```

Comment:
Shall we use some local variables here like 
```suggestion
    [p, q] = ip;
```

Reply:
Changed